Test-№1.1-from-19.10.2022/task2.c

```diff
@@ -0,0 +1,67 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+void swap(int *firstElement, int *secondElement)
+{
+    int temp = *secondElement;
+    *secondElement = *firstElement;
+    *firstElement = temp;
+}
+
+void selectionSort(int *array, int lenOfArray)
+{
+    int minim = 0, pointer = 0;
+    for (int i = 0; i < lenOfArray; ++i)
+    {
+        minim = array[i];
+        pointer = i;
+        for (int j = i + 1; j < lenOfArray; ++j)
+        {
+            if (array[j] < minim)
+            {
+                minim = array[j];
+                pointer = j;
+            }
+        }
+        if (array[i] > minim)

```

Comment:
```suggestion
        if (i != pointer)

```
Проверка, конечно, корректна. Но переусложнена. Так и вырезку элемента массива выполнять не нужно, и алгоритм становится понятнее.